src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ITranslatedQueryOperation.vb

```diff
@@ -0,0 +1,291 @@
+                                    Instance Receiver: 
+                                      IConversionOperation (TryCast: False, Unchecked) (OperationKind.Conversion, Type: System.Collections.Generic.IEnumerable(Of System.Int32), IsImplicit) (Syntax: 'Aggregate y ... t(), Sum(y)')
+                                        Conversion: CommonConversion (Exists: True, IsIdentity: False, IsNumeric: False, IsReference: True, IsUserDefined: False) (MethodSymbol: null)
+                                          (WideningReference)
+                                        Operand: 
+                                          IFlowCaptureReferenceOperation: 1 (OperationKind.FlowCaptureReference, Type: System.Int32(), IsImplicit) (Syntax: 'a')
+                                    Arguments(1):
+                                        IArgumentOperation (ArgumentKind.Explicit, Matching Parameter: selector) (OperationKind.Argument, Type: null, IsImplicit) (Syntax: 'y')
+                                          IDelegateCreationOperation (OperationKind.DelegateCreation, Type: System.Func(Of System.Int32, System.Int32), IsImplicit) (Syntax: 'y')
+                                            Target: 
+                                              IFlowAnonymousFunctionOperation (Symbol: Function (y As System.Int32) As System.Int32) (OperationKind.FlowAnonymousFunction, Type: null, IsImplicit) (Syntax: 'y')
+                                          InConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+                                          OutConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+
+        Next (Regular) Block[B2]
+            Leaving: {R1}
+    
+    {   Function (y As System.Int32) As System.Int32
+    
+        Block[B0#0R1] - Entry
+            Statements (0)
+            Next (Regular) Block[B1#0R1]
+        Block[B1#0R1] - Block
+            Predecessors: [B0#0R1]
+            Statements (0)
+            Next (Return) Block[B2#0R1]
+                IParameterReferenceOperation: y (OperationKind.ParameterReference, Type: System.Int32) (Syntax: 'y')
+        Block[B2#0R1] - Exit
+            Predecessors: [B1#0R1]
+            Statements (0)
+    }
+}
+
+Block[B2] - Exit
+    Predecessors: [B1]
+    Statements (0)
+]]>.Value
+
+            VerifyFlowGraphAndDiagnosticsForTest(Of MethodBlockSyntax)(source, expectedGraph, expectedDiagnostics)
+        End Sub
+
+        <CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)>
+        <Fact()>
+        Public Sub AggregateFlow_02()
+            Dim source = <![CDATA[
+Imports System
+Imports System.Linq
+
+Public Structure C
+
+    Sub M(a As Integer(), b As Integer(), result As object) 'BIND:"Sub M"
+        result = Aggregate y In If(a, b) Into Count(), Sum(y)
```

Comment:
Does the `Into` syntax allow for explicit names for the anonymous type elements?

Reply:
> Does the Into syntax allow for explicit names for the anonymous type elements?

Yes it does.

---
In reply to: [192755825](https://github.com/dotnet/roslyn/pull/27391#discussion_r192755825) [](ancestors = 192755825)

Reply:
Would it make sense to test an aggregate with names then?

Reply:
> Would it make sense to test an aggregate with names then?

Names play no role in the control flow or the shape of the nodes we are going after in this PR. Given the time constraints, we are not doing discovery testing.

---
In reply to: [192882767](https://github.com/dotnet/roslyn/pull/27391#discussion_r192882767) [](ancestors = 192882767)